### PR TITLE
fix TableHandler and ListItemSpan

### PR DESCRIPTION
**Pull request by aslgb**
1. process <th> tag like <td> tag
2. expand table row height on single line height
3. measure list item span margins properly

### CODE REVIEW

**Comment by aslgb** [comment]
@AlexeiVitenko, take a look, please

File: src/main/java/net/nightwhistler/htmlspanner/spans/ListItemSpan.java [hunk 1]
```diff
@@ -33,14 +32,10 @@
...
+
 public class ListItemSpan implements LeadingMarginSpan {
     private final int mNumber;
-
-    private static final int BULLET_RADIUS = 3;
-    private static final int NUMBER_RADIUS = 5;
-
-    //Gap should be about 1em
-    public static final int STANDARD_GAP_WIDTH = HtmlSpanner.HORIZONTAL_EM_WIDTH;
+    private int mWidth;
```
Comment (aliakseivitenka) [inline src/main/java/net/nightwhistler/htmlspanner/spans/ListItemSpan.java:38]: А где инитится mWidth?

Comment (aslgb) [inline src/main/java/net/nightwhistler/htmlspanner/spans/ListItemSpan.java:38]: строка 66, в методе drawLeadingMargin. из документации метода 

> Renders the leading margin. This is called before the margin has been adjusted by the value returned by getLeadingMargin(boolean).

**Comment by aliakseivitenka** [comment]
LGTM

**Pull request merged**